jolt-core/src/jolt/instruction/remu.rs

```diff
@@ -206,10 +206,20 @@ impl<const WORD_SIZE: usize> VirtualInstructionSequence for REMUInstruction<WORD
     }
 
     fn sequence_output(x: u64, y: u64) -> u64 {
-        match WORD_SIZE {
-            32 => (x as u32 % y as u32) as u64,
-            64 => x % y,
-            _ => panic!("Unsupported WORD_SIZE: {}", WORD_SIZE),
+        let quotient = if y == 0 {
+            match WORD_SIZE {
+                32 => u32::MAX as u64,
+                64 => u64::MAX,
+                _ => panic!("Unsupported WORD_SIZE: {}", WORD_SIZE),
+            }
+        } else {
+            x / y
+        };
```

Comment:
let's get rid of this and just move the existing `match` into the `else` block below

Reply:
Sounds good!

Reply:
Fix up